docs/installation.rst

```diff
@@ -23,14 +23,16 @@ Setting up Vagrant
 
      >vagrant up
 
-  Note: If you encounter an error saying "It appears your machine doesn't support NFS, or there is not an
-  adapter to enable NFS on this machine for Vagrant.", then you need to install ``nfs-kernel-server`` using the command:
+  **Troubleshooting**: If you encounter an error saying *"It appears your machine doesn't support NFS, or there is not an adapter to enable NFS on this machine for Vagrant."* there could be two possible causes. You should first install ``nfs-kernel-server`` if it is not already installed, using the command:
 
```

Comment:
So this is kind of wrong. The NFS error is unrelated to this problem and the existing advice is sufficient to fix that, as far as I'm aware... So I'd keep the wording around that, but add a new section about the virtualization error.